index.html

```diff
@@ -1,13 +1,34 @@
 <!DOCTYPE html>
 <html>
+
 <head>
 	<title>JS Onboard Project</title>
 	<script type="text/javascript" charset="utf-8" src="third_party/jquery-2.0.3.min.js"></script>
+	<link rel="stylesheet" href="/Styles.css">
+	<script src="/Interfaces.js"></script>
+	<script src="ApiData.js"></script>
```

Comment:
`ApiData` mentions `GRID_RATIO` which is defined in `app`. So you either have to

* move `ApiData` after `app` (I would not recommend this)
* or move `GRID_RATIO` to `ApiData`
* or create a different file to contain it and pull in that file before `ApiData`.

(The same is true for `ROW_HEIGHT` and `debounce`)